src/Components/Form/ColorForm.jsx

```diff
@@ -0,0 +1,52 @@
+import "./ColorForm.css";
+import { uid } from "uid";
+import ColorInput from "./ColorInput";
+
+export default function ColorForm({
+  onAddColor,
+  initialData = { role: "some color", hex: "#123456", contrastText: "#ffffff" },
+}) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    const newColor = {
+      id: uid(),
+      role: data.role || initialData.role,
+      hex: data.hex || initialData.hex,
+      contrastText: data.contrastText || initialData.contrastText,
+    };
```

Comment:
there is another way to achieve this: 
```js
const newColor = {
  id: uid(),
  ...initialData,
  ...data
}
```
If data contains hex, role or contrast, it will overwrite the default added by initialData.

Reply:
@F-Kirchhoff when adding a new color, but only with input of hex and text, without role name, the card does not show the role name. When writing the code as above it is shown. How could I fix this in your code?